ui/raidboss/data/04-sb/alliance/ridorana_lighthouse.js

```diff
@@ -846,14 +846,14 @@
         'Construct 7': 'Bâtisseur n°7',
         'Construct 7.1': 'Bâtisseur n°7 amélioré',
         'Dark Rain': 'Trombe d\'eau',
-        'Echoes from Time\'s Garden will be sealed off': 'Fermeture du Jardin d\'un autre temps dans',
+        'Echoes from Time\'s Garden': 'Jardin d\'un autre temps dans',
         'Famfrit, The Darkening Cloud': 'Famfrit le Nuage Ténébreux',
         '(?<! )Gigas': 'serviteur de Bélias',
         'Heart of the Dragon': 'cœur de Yiazmat',
         'Missile': 'Missile',
-        'The Cleft of Profaning Wind will be sealed off': 'Fermeture de la corniche des Vents distordants dans',
-        'The Clockwork Coliseum will be sealed off': 'Fermeture de l\'arène de Goug dans',
-        'The Spire\'s Bounds will be sealed off': 'Fermeture de l\'orée des Cieux dans',
+        'The Cleft of Profaning Wind': 'la corniche des Vents distordants dans',
```

Comment:
Same here (remove _dans_) and same for lines 855 & 856.

Reply:
Done, thanks!